lib/xboxrt/Makefile

```diff
@@ -4,6 +4,7 @@ SRCS += \
 	$(NXDK_DIR)/lib/xboxrt/libc_extensions/wchar.c \
 	$(NXDK_DIR)/lib/xboxrt/libc_extensions/stdlib_ext_.c \
 	$(NXDK_DIR)/lib/xboxrt/libc_extensions/string_ext_.c \
+	$(NXDK_DIR)/lib/xboxrt/libc_extensions/wchar_ext_.c \
```

Comment:
nit: I'd move this directly below wchar.c. So if wchar-related errors occur, they are always close to each other in the compilation logs

Reply:
Done.